app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -479,7 +480,7 @@ def alert_build_ems_alarms
     begin
       alarms = MiqAlert.ems_alarms(@edit[:new][:db], @edit[:new][:expression][:options][:ems_id])
     rescue StandardError => bang
-      add_flash(_("Error during '%s': ") % "ems_alarms" << bang.message, :error)
+      add_flash(_("Error during ems_alarms : %{messages}") % {:messages => bang.message}, :error)
```

Comment:
The `ems_alarms` string here seems a bit unfortunate (yes, it was like that before), but I suggest to use just
'alarms' here.